examples/todomvc/src/template.rs

```diff
@@ -0,0 +1,54 @@
+use store::{ItemList, ItemListTrait};
+
+fn escape_html(val: String) -> String {
+    val.replace('&', "&alt;").replace('<', "&lt;").replace('>', "&gt;").replace('"', "&quot").replace('\'', "&#39;")
+}
+
+pub struct Template {}
```

Comment:
Along the lines of showing off crates.io it might be cool to pull in something like handlebars here

Reply:
the 'handlebars' crate seems to add serde which would likely bloat the demo a lot and it looks like maud isn't compiling at the moment. Could this be a follow up bug perhaps? Unless you have other suggestions?